src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.nuspec

```diff
@@ -2,6 +2,9 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata>
     $CommonMetadataElements$
+    <dependencies>
+      <dependency id="Microsoft.AspNetCore.Components.WebAssembly.DebugProxy" version="$MicrosoftAspNetCoreComponentsWebAssemblyDebugProxyPackageVersion$" include="all" />
```

Comment:
Is there a reason it couldn't pick up the proxy from WebAssembly.Server? Why does it need the package reference?